src/libvirt-dbus.js

```diff
@@ -1483,6 +1484,21 @@ export function getDomainCapabilities(connectionName, arch, model) {
     return call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'GetDomainCapabilities', ['', arch, model, '', 0], { timeout, type: 'ssssu' });
 }
 
+export function getPoolCapabilities({ connectionName }) {
+    // TODO: replace with D-Bus API once available https://bugzilla.redhat.com/show_bug.cgi?id=1986321
+    const opts = { err: "message", environ: ['LC_ALL=C'] };
+    if (connectionName === 'system')
+        opts.superuser = 'try';
+
+    return cockpit.spawn(
+        ["virsh", "-c", "qemu:///" + connectionName, "pool-capabilities"],
+        opts
+    ).then(poolCapabilities => parsePoolCapabilities(poolCapabilities), ex => {
+        console.warn('virsh pool-capabilities failed:', ex.toString());
+        return cockpit.reject(ex);
```

Comment:
Let's not turn JS Promises into the obsolete cockpit jQuery promises. However, I see that we already do this in another place, I'll just send a follow-up after this.